src/ScriptEditor/ScriptEditor.ts

```diff
@@ -100,6 +103,25 @@ export class ScriptEditor {
           2686,
         ],
       });
+      // Sync all javascript and typescript text models to both language servers.
+      {
```

Comment:
What's up with this extra block just hanging out here? Do we need this scope for some reason?

Reply:
It is to scope the comment.

Reply:
I think this's the first time I've ever seen that. 

It's pretty clear what the scope of the comment is without being that explicit; more importantly, there is no harm done if someone misinterprets the scope slightly.
You can use an extra newline to indicate "this is not directly part of the next comment," and then you don't need this while extra level of indentation.